requirements.txt

```diff
@@ -1,5 +1,5 @@
 apache-airflow == 2.10.0
 requests == 2.32.3
-setuptools == 65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
+setuptools == 70.0.0 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
with pyptoject.toml, requirements.txt is not needed. Still keeping it for backward compatibility of different deployment processes.